packages/ListBox/test/helpers/toggleHelpers.js

```diff
@@ -6,5 +6,4 @@ export function openLazyDropDown() {
 
 export function toggleDropdown() {
   cy.get(selectors.trigger).click();
-  cy.get(selectors.popover).should("be.visible");
```

Comment:
This was failing for me now, but then i also realized this is a strange place for this call to be made as it is a toggle action, so it may be  closing rather than opening the popover, so I don't think it belongs here, and removed